text/0233-opentelemetry-sandbox.md

```diff
@@ -0,0 +1,89 @@
+# OpenTelemetry Sandbox
+
+The OpenTelemetry Sandbox is a place under OpenTelemetry's governance where the community can collaborate and experiment on projects that still aren't mature enough to be accepted as part of the official OpenTelemetry project.
+
+## Motivation
+
+Over the history of OpenTelemetry, there have been situations where people came to our community proposing interesting ideas to be adopted. There has also been vendors offering code donations to the project, some of which are now mostly unmaintained.
+
+As a possible solution to this, this OTEP proposes a new GitHub organization, [opentelemetry-sandbox](https://github.com/opentelemetry-sandbox). This organization will host projects until there's confidence that they have a healthy community behind them. They would also serve as a neutral place for the community to conduct experiments.
+
+The advantage of a sandbox organization is that OpenTelemetry's governance rules can be applied, making sure it’s an inclusive place for people to collaborate while keeping the reputation of the OpenTelemetry project as a whole untouched, given that it would be clear that OpenTelemetry doesn’t officially support projects within the sandbox.
```

Comment:
> governance rules can be applied

What specifically does that mean? That TC is expected to arbitrate technical disagreements? Even that does not remove the need for internal governance of the proposed project.

Reply:
I had CoC violations in mind when I wrote this but didn't want to limit the scope to that, as I can't foresee the problems we'll have.

> That TC is expected to arbitrate technical disagreements?

Great question, I'll add this to the open questions.